ppml/base/bash.manifest.template

```diff
@@ -36,14 +36,12 @@ loader.env.SPARK_SCALA_VERSION = "2.12"
 loader.env.SPARK_HOME = "{{ spark_home }}"
 loader.env.SPARK_MASTER_OPTS = "'-Dspark.worker.timeout=60'"
 
-sgx.ioctl_structs.ifconf  =  [ {name = "ifc_len",size = 4, align = 4, type= "inout" }, {size = 4, type= "inout"}, { ptr=[ {name="ifcu_req", size= "ifc_len", type="in"} ] } ]
-sgx.allowed_ioctls.io1.request = 0x8912
-sgx.allowed_ioctls.io1.struct = "ifconf"
-
-sgx.ioctl_structs.ifreq = [ {  name ="ifreq", size=40, type="inout"} ]
-sgx.allowed_ioctls.io2.request = 0x8927
-sgx.allowed_ioctls.io2.struct = "ifreq"
-
+sgx.ioctl_structs.ifconf = [ {name = "ifc_len",size = 4, direction = "inout" }, {size = 4}, { ptr=[ {size= "ifc_len", direction = "in"} ] } ]
```

Comment:
what is the function of the appended parameters? Will they influence performance of other applications? If yes, we can ADD a new manifest only in trusted-machine-learning instead of base.